BookStore/controllers/product.js

```diff
@@ -0,0 +1,62 @@
+const formidable = require('formidable');
+const _ = require('lodash');
```

Comment:
Why have you imported `lodash` in but then not used it anywhere? If it's not being used yet, you should delete it from this PR

Reply:
we just  used it. come on jamie